guides/common/modules/proc_creating-hosts-on-vmware.adoc

```diff
@@ -52,7 +52,11 @@ endif::[]
 Modify these settings to suit your requirements.
 ifdef::satellite,orcharhino[]
 . Click the *Parameters* tab and ensure that a parameter exists that provides an activation key.
-If not, add an activation key.
+If a parameter does not exist, click *+ Add Parameter*.
++
+* In the field *Name*, enter *kt_activation_keys*.
+* In the field *Value*, enter the name of the *Activation Key* used to register the Content Hosts.
++
 endif::[]
```

Comment:
```suggestion
.. In the *Name* field, enter *kt_activation_keys*.
.. In the *Value* field, enter the name of the activation key used to register the Content Hosts.
endif::[]
```

Reply:
Even this has point. What style guide has to say?

Reply:
Haven't found anything in the quick reference, @maximiliankolb what do you think?

Reply:
![image](https://user-images.githubusercontent.com/101559972/211569807-ad377b16-394e-4f08-a3a3-b0da2834d93d.png)

This is what Style Guide has to say.

Reply:
I think so far, we're avoiding substeps and bullet points in procedures and keep lists as flat as possible. I lean towards the "procedure" example.

Reply:
Kept flat without further formatting :+1: